ui/app/selectors/selectors.js

```diff
@@ -475,3 +481,11 @@ export function getIsSwapsChain(state) {
   const chainId = getCurrentChainId(state);
   return ALLOWED_SWAPS_CHAIN_IDS[chainId];
 }
+
+export function getPrimaryTokenImage(state) {
+  const chainId = getCurrentChainId(state);
```

Comment:
Perhaps it would be better to use `nativeCurrency` here instead of the `chainId`? That's what this seems to be for - a representation of the native currency.

This would let us fix this for _all other_ custom networks as well, by displaying _nothing_ if we don't recognize the native currency. We'd ideally want to show the ETH symbol only of the native currency is ETH, rather than as a fallback all of the time.

Reply:
Good idea!